internal/libs/cciptypeutil/seqnumrange.go

```diff
@@ -0,0 +1,21 @@
+package cciptypeutil
+
+import "github.com/smartcontractkit/chainlink-common/pkg/types/ccipocr3"
+
+// SeqNumRangeLimit limits the range to n elements by truncating the end if necessary.
+func SeqNumRangeLimit(rng ccipocr3.SeqNumRange, n uint64) ccipocr3.SeqNumRange {
```

Comment:
nit: could add a `.Limit` function to the SeqNumRange type in chainlink-common.

Reply:
Added